seong-geun/app.js

```diff
@@ -0,0 +1,46 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+const dotenv = require("dotenv");
+dotenv.config();
```

Comment:
```suggestion
require("dotenv").config();
```
두줄로 작성된 기존 코드를 변수 선언 없이 한줄로써 작성될 수 있습니다.